docs/dev/setup.sh

```diff
@@ -7,11 +7,23 @@ if [ -z "$stream_backend" ] || (echo "stan jetstream" | grep -v -q "$stream_back
 fi
 echo "Using $stream_backend"
 
+COMPOSE_CMD='docker-compose'
+
+DCLIENT_VERSION=$(docker version -f '{{.Client.Version}}')
+if echo $DCLIENT_VERSION | grep -q '^2' ; then
+  COMPOSE_CMD='docker compose'
```

Comment:
We could, but when Docker Desktop removes `docker-compose` completely (it's now just deprecated), this will break.

Reply:
I think it makes sense to add a small doc here about how to get `docker compose` working on EC2, whether it's here or in a followup, using the method we determined on our zoom call to make it behave similarly to other OSes. (Especially since the installation of docker-compose is already done via external, non-rpm-package means on Amazon Linux).